grid-packages/ag-grid-docs/documentation/src/pages/PolicyPages.module.scss

```
@@ -0,0 +1,70 @@
+@use '../design-system' as *;
+
+.policyPage {
+    margin-top: $size-6;
+    margin-bottom: calc($size-6 * 4);
+
+    nav {
+        margin-top: $size-4;
+        margin-bottom: $size-4;
+    }
+
+    nav li:not(:last-child) {
+        margin-bottom: $size-half;
+    }
+
+    hr {
+        margin-bottom: $size-1;
+    }
+
+    @media screen and (min-width: 1400px) {
```

Should this use `$max-page-width`?

```suggestion
    @media screen and (min-width: $max-page-width) {
```